ui-components/UserUpdateForm.jsx

```diff
@@ -0,0 +1,380 @@
+      ></TextField>
+      <TextField
+        label="Last name"
+        isRequired={false}
+        isReadOnly={false}
+        value={lastName}
+        onChange={(e) => {
+          let { value } = e.target;
+          if (onChange) {
+            const modelFields = {
+              firstName,
+              lastName: value,
+              email,
+              institution,
+              allergies,
+              checkedIn,
+            };
+            const result = onChange(modelFields);
+            value = result?.lastName ?? value;
+          }
+          if (errors.lastName?.hasError) {
+            runValidationTasks("lastName", value);
+          }
+          setLastName(value);
+        }}
+        onBlur={() => runValidationTasks("lastName", lastName)}
+        errorMessage={errors.lastName?.errorMessage}
+        hasError={errors.lastName?.hasError}
+        {...getOverrideProps(overrides, "lastName")}
+      ></TextField>
+      <TextField
+        label="Email"
+        isRequired={false}
+        isReadOnly={false}
+        value={email}
+        onChange={(e) => {
+          let { value } = e.target;
+          if (onChange) {
+            const modelFields = {
+              firstName,
+              lastName,
+              email: value,
+              institution,
+              allergies,
+              checkedIn,
+            };
+            const result = onChange(modelFields);
+            value = result?.email ?? value;
+          }
+          if (errors.email?.hasError) {
+            runValidationTasks("email", value);
+          }
+          setEmail(value);
+        }}
+        onBlur={() => runValidationTasks("email", email)}
+        errorMessage={errors.email?.errorMessage}
+        hasError={errors.email?.hasError}
+        {...getOverrideProps(overrides, "email")}
+      ></TextField>
+      <TextField
+        label="Institution"
+        isRequired={false}
+        isReadOnly={false}
+        value={institution}
+        onChange={(e) => {
+          let { value } = e.target;
+          if (onChange) {
+            const modelFields = {
+              firstName,
+              lastName,
+              email,
+              institution: value,
+              allergies,
+              checkedIn,
+            };
+            const result = onChange(modelFields);
+            value = result?.institution ?? value;
+          }
+          if (errors.institution?.hasError) {
+            runValidationTasks("institution", value);
+          }
+          setInstitution(value);
+        }}
+        onBlur={() => runValidationTasks("institution", institution)}
+        errorMessage={errors.institution?.errorMessage}
+        hasError={errors.institution?.hasError}
+        {...getOverrideProps(overrides, "institution")}
+      ></TextField>
+      <TextField
+        label="Allergies"
+        isRequired={false}
+        isReadOnly={false}
+        value={allergies}
+        onChange={(e) => {
```

Comment:
same as above

Reply:
These are auto-generated and we will deleted these UI-forms since our front-end devs will be making these up to follow the rest of the site.